cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -240,3 +241,21 @@ def test_import_report(appliance):
     response, = appliance.rest_api.collections.reports.action.execute_action("import", data)
     assert_response(appliance)
     assert response['message'] == 'Skipping Report (already in DB): [{}]'.format(menu_name)
+
+
+@pytest.mark.sauce
+@pytest.mark.tier(3)
+@test_requirements.report
+def test_reports_delete_saved_report(custom_report_values, appliance, request):
+    """The test case selects reports from the Saved Reports list and deletes them.
+    """
+    report = appliance.collections.reports.create(**custom_report_values)
+    request.addfinalizer(report.delete)
+    report.queue(wait_for_finish=True)
+    view = navigate_to(appliance.collections.saved_reports, 'All')
+    # iterates through every row and checks if the 'Name' column matches the given value
+    for row in view.table.rows():
+        if row.name.text == report.title:
+            row[0].check()
+    view.configuration.item_select(
+        item='Delete selected Saved Reports', handle_alert=True)
```

Comment:
**REQUIRED:** This is a great looking test, but there are a few things we should address.
1) The most important, is that though you invoke the button, you never check to see if the saved report is actually gone. We should at least refresh the list and ensure it is no longer there.
2) We should investigate making the saved report have a method to "delete". It is possible people would like to reuse this behaviour.
The first of these points is REQUIRED. The second is OPTIONAL, but should have an issue raised against it.